setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup(name='flooddrake',
+      description='Firedrake DG shallow water model with wetting and drying for flood simulation',
+      url='https://github.com/firedrakeproject/flooddrake',
+      author='Alastair C. A. Gregory, Colin. J. Cotter',
+      author_email='alsgregory@aol.com',
+      license='Imperial College London',
```

Comment:
I've never done one before so copied another. I may have actually thought that was institution! So yes I think I should of left it as that - sorry!